pkg/windows/create_build_env.ps1

```diff
@@ -58,6 +58,7 @@ if (!($7zip)) {
 if (Test-Path .\salt) {
     Remove-Item -Recurse -Force salt
 }
+Push-Location
```

Comment:
this statement was lost in https://github.com/hubblestack/hubble/commit/b3c3ef0d03d4fe6bcf0946a9e938d843ef643150#diff-dfb17bd604fab45bce0fb57b3e965f93
code is not able to find salt directory in line number 66/67 ( Push-Location salt )

Reply:
I am unsure what this does, since you aren't specifying a directory.  I think what needs to be done here is keep this removed, then change line 63 from set-location to push-location